ui/raidboss/data/04-sb/raid/o7s.js

```diff
@@ -37,11 +38,12 @@
       },
     },
     {
-      regex: / 1A:\y{ObjectId}:Guardian gains the effect of Dadaluma Simulation/,
-      regexDe: / 1A:\y{ObjectId}:Wächter gains the effect of Dadarma-Kampfprogramm/,
-      regexFr: / 1A:\y{ObjectId}:Gardien gains the effect of Programme Dadaluma/,
-      regexJa: / 1A:\y{ObjectId}:ガーディアン gains the effect of ダダルマー・プログラム/,
-      regexKo: / 1A:\y{ObjectId}:가디언 gains the effect of 다다루마 프로그램/,
+      regex: Regexes.gainsEffect({ effect: 'Dadaluma Simulation', capture: false }),
```

Comment:
This needs a `target: Guardian` (and a few others in this file)

Reply:
ok added Guardian to /hopefully) all entries